src/jailer/src/resource_limits.rs

```diff
@@ -187,4 +187,37 @@ mod tests {
         assert_eq!(rlim.rlim_cur, new_limit);
         assert_eq!(rlim.rlim_max, new_limit);
     }
+
+    #[test]
+    fn test_install() {
+        // Setup the resource limits
+        let mut rlimits = ResourceLimits::default();
+        let new_limit = 100;
```

Comment:
You might have to choose a bigger value here, the tests fail with `SIGXFSZ`, which seems to be caused by some file being larger than the allowed limit: https://en.wikipedia.org/wiki/Signal_(IPC)#SIGXFSZ

Reply:
ah I see. do you know the largest file size used by those tests? also I can I run those locally?